### ISSUE TITLE
Display saved date for gifts

### ISSUE THREAD

**Issue by mgrhm** [monicahq/monica#5080]
The ability to add a date to a gift was added by @sktlmarat in #4909, and works.  Monica correctly saves and stores the date of the gift.  Unfortunately, it doesn't by default *display* this date in a useful fashion -- it's hidden and only visible when [switching into 'Edit' mode](https://github.com/monicahq/monica/issues/4909#issuecomment-803639269).

This information is too useful to hide by default.  I tried to add this myself by editing `Gifts.vue`, but I don't know any Javascript and couldn't make head nor tails of it.  I wasn't able to add it in myself.

I've made a mockup of where I *think* would be a good place for it, and attached it to this issue.  (It's on the right hand side of the gift's title bar.)

![Screenshot 2021-04-19 at 17 29 03](https://user-images.githubusercontent.com/21129642/115271912-beb7fe80-a135-11eb-9199-63bcdec713d2.png)

**Comment by sktlmarat** [comment]
Hi @mgrhm ,

Can you make a PR with the ready markup? I will then make a few corrections to your code. I guess it won't require too much code.

**Comment by mgrhm** [comment]
Hi @sktlmarat.  I really have absolutely no idea where to start with this -- I know a bit of HTML, a bit of CSS, but no PHP or Javascript at all.

I've looked at the code in `resources/js/components/people/gifts/Gifts.vue`.  I have no idea at all how to add this.  From what I can work out, I *think* the code would need to be added somewhere between lines 57 and 61:

      <div v-for="gift in filteredGifts" :key="gift.id" v-cy-name="'gift-item-' + gift.id" class="ba b--gray-monica mb3 br2">
        <gift v-if="!gift.edit"
              :gift="gift"
              @update="($event) => { updateList($event) }"
        >

I’m really not sure if this is right, though, and I'm grateful for any input you have.

**Comment by sktlmarat** [comment]
@mgrhm, I need css/html of this. 
![115271912-beb7fe80-a135-11eb-9199-63bcdec713d2](https://user-images.githubusercontent.com/40805789/115375886-0602c580-a1f0-11eb-93ee-05e1a53ff4d2.png)

**Comment by mgrhm** [comment]
Sorry -- I don't actually have CSS or HTML for that.  It's just an image that I edited using Paintbrush to add the date onto a screenshot.

I've spent some time this afternoon trying to work out which CSS classes would apply, but I have genuinely no idea -- I can't make heads nor tails of either the generated output or the `.vue` input files.  I'm sorry I can't be of more help.  This is just beyond my very limited skills.
